src/components/TodoItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const completedStyle = {
+  fontStyle: 'italic',
+  color: '#595959',
+  opacity: 0.4,
+  textDecoration: 'line-through',
+};
+
+const TodoItem = (props) => {
+  const { todo } = props;
+  const { handlechangeProps } = props;
+  const { handledeleteProps } = props;
```

Comment:
- [ ]  You can join this 3 const in one line like this: const `{ todo, handlechangeProps, handledeleteProps } = props;`

Reply:
noted 😉 I will make the change.